.goreleaser.yml

```diff
@@ -0,0 +1,56 @@
+project_name: cyberark-sidecar-injector
+
+builds:
+- id: cyberark-sidecar-injector-linux
+  main: ./cmd/sidecar-injector
+  env:
+  - CGO_ENABLED=0
+  # Tag 'netgo' is a Go build tag that ensures a pure Go networking stack
+  # in the resulting binary instead of using the default host's stack to
+  # ensure a fully static artifact that has no dependencies.
+  flags:
+  - -tags=netgo
+  goos:
+  - linux
+  goarch:
+  - amd64
+  # The `Tag` override is there to provide the git commit information in the
```

Comment:
Does this "`Tag` override is there..." comment belong below at Line 47?

Reply:
Good catch. Too much copy and pasting